modules/sr/robot/randomizer.py

```diff
@@ -21,3 +24,21 @@ def add_jitter(
     random_range = actual_value * (random_range_percent / float(100))
     new_value = actual_value + random.uniform(-random_range, random_range)
     return type(actual_value)(max(min_possible, min(new_value, max_possible)))
+
+
+def add_independent_jitter(
+    actual_value: T,
+    min_possible: T,
+    max_possible: T,
+    std_dev_percent: float =  # % of full scale value
+    DEFAULT_RANDOM_INDEPENDENT_RANGE_PERCENT,
+    can_wrap: bool = False,
+) -> T:
+    value_range = max_possible - min_possible
+    random_range = value_range * (std_dev_percent / float(100))
```

Comment:
I wonder if the difference should be fixed instead? ie taking as an argument the +/- value explicitly.

Reply:
We could, although we'd want to remove the default value in that case.

Reply:
I think that's ok, forces us to think about it. It also removes the magic number currently being passed, which is a little terse

Reply:
Essentially we'd be changing from a value relative to full scale to an absolute value. The upside of dealing in relative values is that it is a unit-less quantity so we don't need to bother with converting our constant between degrees and radians.